packages/ui/src/components/input-field/input-field.tsx

```diff
@@ -0,0 +1,87 @@
+import clsx from 'clsx';
+import { AnimatePresence, m } from 'framer-motion';
+import React from 'react';
+
+import { easeInOutOpacity } from '../animation';
+import { Text } from '../text';
+import { SelectInputProps } from './select-input';
+import { TextInputProps } from './text-input';
+
+export type InputFieldProps = {
+  children: React.ReactElement<TextInputProps | SelectInputProps>;
+  label?: string;
+  placeholder?: string;
+  errorMessage?: string;
+  hintText?: string;
+  disabled?: boolean;
+  className?: string;
+  /**
+   * @default 'vertical'
+   */
+  layout?: 'vertical' | 'horizontal';
+};
+
+export function InputField({
```

Comment:
Will you refactor existing input label components with the new component?

Reply:
Not sure what's the `input label` component?